SetupNode/scripts/prepare_remote_machine.py

```diff
@@ -104,54 +128,55 @@ def extract_name_and_ip(name, return_data):
 
 
 def prepare_machine_to_docker_node(ip):
-    with open('hostname', mode='w') as file:
-        file.write(os.environ['REMOTE_HOST_NAME'])
-        file.close()
-
+    _, stdout, _ = client.exec_command('hostname')
+    hostname = (stdout.read()).decode('ascii').replace('\n', '')
     with open('hosts', mode='w') as file:
```

Comment:
Let's remove update to /etc/hosts
and add documentation that hostnames should be on DNS